README.md

```diff
@@ -148,7 +273,7 @@ Configuration can also be provided by a config file (`--config` or `CONFIG`), or
 
 You can also have an external provider create a secret for you in the namespace before deploying the chart with helm. If the secret is pre-provisioned, replace the `agentToken` and `graphqlToken` arguments with:
 
-```bash
+```console
```

Comment:
I noticed you changed these from `bash` or `shell` to `console`, but they don't have shell prompts - is there a reason for the change?

Conversely, I think the example console sessions you added *should* be formatted as `console`, and include `$` prompts before the example commands run (see my last comment).

Reply:
I made this change for consistency but looking at the linguist documentation I now see there are some subtle differences. I'll review these to make sure the usage is correct for each case and add $ prompts to the example sessions to improve readability and consistency.